// react-cssfx-loading/lib/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.TwinSpin = exports.SpinStretch = exports.Spin = exports.Ring = exports.Messaging = exports.Hypnosis = exports.FlippingSquare = exports.FillingBottle = exports.FadingDots = exports.FadingBalls = exports.Coin = exports.CircularProgress = exports.BouncingBalls = exports.BarWave = void 0;
var BarWave_1 = require("./BarWave");
Object.defineProperty(exports, "BarWave", { enumerable: true, get: function () { return __importDefault(BarWave_1).default; } });
var BouncingBalls_1 = require("./BouncingBalls");
Object.defineProperty(exports, "BouncingBalls", { enumerable: true, get: function () { return __importDefault(BouncingBalls_1).default; } });
var CircularProgress_1 = require("./CircularProgress");
Object.defineProperty(exports, "CircularProgress", { enumerable: true, get: function () { return __importDefault(CircularProgress_1).default; } });
var Coin_1 = require("./Coin");
Object.defineProperty(exports, "Coin", { enumerable: true, get: function () { return __importDefault(Coin_1).default; } });
var FadingBalls_1 = require("./FadingBalls");
Object.defineProperty(exports, "FadingBalls", { enumerable: true, get: function () { return __importDefault(FadingBalls_1).default; } });
var FadingDots_1 = require("./FadingDots");
Object.defineProperty(exports, "FadingDots", { enumerable: true, get: function () { return __importDefault(FadingDots_1).default; } });
var FillingBottle_1 = require("./FillingBottle");
Object.defineProperty(exports, "FillingBottle", { enumerable: true, get: function () { return __importDefault(FillingBottle_1).default; } });
var FlippingSquare_1 = require("./FlippingSquare");
Object.defineProperty(exports, "FlippingSquare", { enumerable: true, get: function () { return __importDefault(FlippingSquare_1).default; } });
var Hypnosis_1 = require("./Hypnosis");
Object.defineProperty(exports, "Hypnosis", { enumerable: true, get: function () { return __importDefault(Hypnosis_1).default; } });
var Messaging_1 = require("./Messaging");
Object.defineProperty(exports, "Messaging", { enumerable: true, get: function () { return __importDefault(Messaging_1).default; } });
var Ring_1 = require("./Ring");
Object.defineProperty(exports, "Ring", { enumerable: true, get: function () { return __importDefault(Ring_1).default; } });
var Spin_1 = require("./Spin");
Object.defineProperty(exports, "Spin", { enumerable: true, get: function () { return __importDefault(Spin_1).default; } });
var SpinStretch_1 = require("./SpinStretch");
Object.defineProperty(exports, "SpinStretch", { enumerable: true, get: function () { return __importDefault(SpinStretch_1).default; } });
var TwinSpin_1 = require("./TwinSpin");
Object.defineProperty(exports, "TwinSpin", { enumerable: true, get: function () { return __importDefault(TwinSpin_1).default; } });
